src/oop/special_class_method/medium/q6/Scissors.java

```diff
@@ -0,0 +1,32 @@
+package oop.special_class_method.medium.q6;
+
+public class Scissors {
+    private boolean isSharp;
+    private boolean outOfReach;
+
+    public Scissors(boolean isp, boolean cct) {
```

Comment:
since we give this constructor to the user, I feel like cct should get changed to something more relevant to outOfReach